### PR TITLE
Pull request for Game Logic

### PR DESCRIPTION
**Pull request by phfdonda**
The Game Logic was completed. The game:
 - evaluates if the player has marked a sequence of three numbers, be that in diagonal, horizontal or vertical alignment. The game ends in victory for the player that does so. If none is able to do it until all numbers are chosen, then the game is finished as a tie.
- takes turns for the players, prompts for their numbers and checks for attempts of cheating.
- doesn't allow for empty names nor two equal names for the players.
- has most classes kept in the lib folder, except those that retrieve input from the user or that are responsible for printing on the screen in general.
- allows the player the opportunity to skip the rules if they already know them, or to read them if they want.
- has no attr_accessors used in its code.
- has classes that follow the single responsibility rule.

Thank you for your review! I'm very excited about this project, and I hope you like it! Please, let me know if you find any bugs, or if any of these assessments I have made about my project are wrong! Thank you again!

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    @press_enter_to_continuer = clear_input
+  end
+
+  def intro
+    system 'clear'
+    make_space
+    pretty_print(@pretty_intro, ' ')
+  end
+
+  def rules?
```
Comment (Forison) [inline lib/game.rb:60]: This method doesn't seem to return a boolean why attach ```?``` remember the convention in Ruby is that only a method that returns boolean is written this way.

File: lib/table.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    if ind.zero?
+      2
+    elsif ind == 2
+      0
+    else
+      ind
+    end
+  end
+
+  def check_for_seq(position)
```
Comment (Forison) [inline lib/table.rb:27]: This method seems to return a boolean attach``` ?``` remember the convention in Ruby is that method that returns boolean is written this way.

**Review by Forison — APPROVED** [review]
## Status: Approved💚
# 🙋‍♂️ Pedro
I like your pull request🙂, your code is divided into different classes. You try to also implement almost all the basic concepts in Ruby OOP.
I like the creativity you have exhibited in the game implementation. ie when the name is selected and when the same name is selected etc...
Here is how we can even make the project better than it is now. 
The following are optional, but useful in my opinion 
* [ ] When a result is declared allow a user to select if they want to restart the game or not rather than ending the game abruptly.
* [ ] You need to take care of the merge conflict. 
* [ ] Linters must not be edited, it must be maintained as configured by Microverse. 
* [ ] Check the comments below to improve your code.
### _That is it kindly consider these recommendations.
Happy coding
Addo Forison

File: lib/game.rb (hunk 1 above)
Comment (phfdonda) [reply lib/game.rb:60]: Really? I didn't know that! Thank you very much for this lesson! I will change that!

File: lib/table.rb (hunk 2 above)
Comment (phfdonda) [reply lib/table.rb:27]: Again, thank you for this lesson! I will correct that!

**Pull request merged**